File: URI/2758.py
en1 = input().split();
en2 = input().split();

signA = '' if(en1[0][0]!='-') else '-'
signB = '' if(en1[1][0]!='-') else '-'
signC = '' if(en2[0][0]!='-') else '-'
signD = '' if(en2[1][0]!='-') else '-'

a, b = [abs(float(x)) for x in en1]
c, d = [abs(float(x)) for x in en2]

print("A = ",signA,"%f, B = "%a,signB,"%f"%(b),sep='')
print("C = ",signC,"%f, D = "%c,signD,"%f"%(d),sep='')

print("A = ",signA,"%.1f, B = "%a,signB,"%.1f"%(b),sep='')
print("C = ",signC,"%.1f, D = "%c,signD,"%.1f"%(d),sep='')

print("A = ",signA,"%.2f, B = "%a,signB,"%.2f"%(b),sep='')
print("C = ",signC,"%.2f, D = "%c,signD,"%.2f"%(d),sep='')

print("A = ",signA,"%.3f, B = "%a,signB,"%.3f"%(b),sep='')
print("C = ",signC,"%.3f, D = "%c,signD,"%.3f"%(d),sep='')

s1 = ("A = "+signA+"%.3e, B = "%a+signB+"%.3e"%(b)).upper()
s2 = ("C = "+signC+"%.3e, D = "%c+signD+"%.3e"%(d)).upper()
print(s1)
print(s2)

print("A = ",signA,"%d, B = "%int(a),signB,"%d"%int(b),sep='')
print("C = ",signC,"%d, D = "%int(c),signD,"%d"%int(d),sep='')

File: URI/2758.cpp
#include<iostream>
#include<string>
#include<cstdio>
#include<cmath>

using namespace std;

int main(){
	string en1, en2, en3, en4;
	string s1, s2;
	float a, b;
	double c, d;	
	char signA, signB, signD, signC;

	cin >> en1 >> en2;
	cin >> en3 >> en4;

	signA = (en1[0]!='-') ? '\0' : '-';
	signB = (en2[0]!='-') ? '\0' : '-';
	signC = (en3[0]!='-') ? '\0' : '-';
	signD = (en4[0]!='-') ? '\0' : '-';

	a = stof(en1);
	b = stof(en2);
	c = stod(en3);
	d = stod(en4);

	printf("A = %f, B = %f\n",a,b);
	printf("C = %lf, D = %lf\n",c,d);

	printf("A = %.1f, B = %.1f\n",a,b);
	printf("C = %.1lf, D = %.1lf\n",c,d);

	printf("A = %.2f, B = %.2f\n",a,b);
	printf("C = %.2lf, D = %.2lf\n",c,d);

	printf("A = %.3f, B = %.3f\n",a,b);
	printf("C = %.3lf, D = %.3lf\n",c,d);

	printf("A = %.3E, B = %.3E\n",a,b);
	printf("C = %.3E, D = %.3E\n",c,d);

	printf("A = %.f, B = %.f\n",a,b);
	printf("C = %.lf, D = %.lf\n",c,d);
}
